openstates/la/__init__.py

```diff
@@ -22,11 +22,12 @@
         {
             "name": "2012-2015",
             "start_year": 2012,
-            "end_year": 2015,
+            "end_year": 2016,
```

Comment:
```diff
-            "end_year": 2015,
+            "end_year": 2016,
```

I seem to recall LA being a max of 60 calendar days starting sometime in April,
so I think the end year here is 2015. I'll correct this before I merge.